### PR TITLE
Add an option to determine the number of jobs to run simultaneously

### LINKED ISSUE #12: Add jobs option
Currently, the miner can run only one job because it stops the previous job when the new job comes.
I propose a `jobs` option which determines the number of jobs at the same time.

### PR DESCRIPTION
**Pull request by sgkim126**
It fixes #12

### CODE REVIEW

File: src/worker/mod.rs [hunk 1]
```diff
@@ -30,13 +30,15 @@ use super::message::Job;
...
     let submit_port = config.submitting_port();
+    let jobs = config.jobs();
     let addr = ([127, 0, 0, 1], listen_port).into();
 
     let server = Server::bind(&addr)
         .serve(move || service_fn(move |req| get_work(config, req)))
         .map_err(|e| error!("server error: {}", e));
     info!("Server started, listening on {:?}", addr);
     info!("It will submit to 127.0.0.1:{}", submit_port);
+    info!("The maximum number of jobs at the same time is {}", jobs);
```
Comment (kseo) [inline src/worker/mod.rs]: The maximum number of parallel jobs is

**Review by kseo — CHANGES_REQUESTED** [review]

File: src/worker/work.rs [hunk 2]
```diff
@@ -28,22 +28,22 @@ use super::Worker;
 
 static JOB_ID: AtomicUsize = AtomicUsize::new(0);
 
-pub fn spawn_worker(hash: H256, target: U256, worker: Box<Worker>, submit_port: u16) {
+pub fn spawn_worker(hash: H256, target: U256, worker: Box<Worker>, submit_port: u16, jobs: usize) {
```
Comment (remagpie) [inline src/worker/work.rs:31]: It seems `jobs` and `worker` both comes from `Config` object.
How about passing `config: Config` instead?

Comment (sgkim126) [reply src/worker/work.rs:31]: I agree with you. I'll change it in the next patch.

**Review by kseo — APPROVED** [review]
LGTM

**Pull request merged**